app/assets/javascripts/discourse/app/components/sidebar/user/custom-sections.hbs

```diff
@@ -17,15 +17,11 @@
             @prefixValue={{link.icon}}
             @href={{link.value}}
             @class={{link.linkDragCss}}
```

Comment:
canReorder was used to remove functionality from touch screens. Not required anymore